tests/integration/test_endpoints.py

```diff
@@ -26,7 +29,12 @@
 ]
 
 
-def test_simulate_wrong_id(client):
+def test_simulate_wrong_id(monkeypatch, client):
+    # Mock `requests` to skip the external API request
+    def mock_response(*args, **kwargs):
+        Response = namedtuple("Response", ["status_code"])
```

Comment:
As a suggestion: Move the definition of `Response` outside of the function and make `mock_response` a lambda.